docs/io/fits/appendix/faq.rst

```diff
@@ -52,19 +52,14 @@ bases is non-trivial.  The second is that there are many features of Astropy
 greatly.  Since PyFITS is already integrated into Astropy, it makes more sense
 to continue development there rather than make Astropy a dependency of PyFITS.
 
-PyFITS' current primary developer and active maintainer is `Erik Bray`_, though
-patch submissions are welcome from anyone.  PyFITS is now primarily developed
-in a Git repository for ease of merging to and from Astropy.  Patches and issue
-reports can be posted to the `GitHub project`_ for PyFITS, or for Astropy.
+PyFITS' past primary developer and active maintainer was `Erik Bray`_.  There
```

Comment:
While you are at it, you can remove the link to @embray's email, as it is no longer valid. I think just mentioning the name without contact info at this point is fine.